CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contribution Guidelines
+
+Thank you for your interest in contributing to Unity Robotics! To facilitate your 
+contributions, we've outlined a brief set of guidelines to ensure that your extensions 
+can be easily integrated.
+
+## Communication
+
+First, please read through our
+[code of conduct](CODE_OF_CONDUCT.md),
+as we expect all our contributors to follow it.
+
+Second, before starting on a project that you intend to contribute to any of our
+Unity Robotics packages or tutorials, we **strongly** recommend posting on the repository's
+[Issues page](https://github.com/Unity-Technologies/URDF-Importer/issues) and
+briefly outlining the changes you plan to make. This will enable us to provide
+some context that may be helpful for you. This could range from advice and
+feedback on how to optimally perform your changes or reasons for not doing it.
+
+Lastly, if you're looking for input on what to contribute, feel free to reach
+out to our team directly at [unity-robotics@unity3d.com](mailto:unity-robotics@unity3d.com) 
```

Comment:
I'm not sure if I just missed this before... but I think we need to be more conservative about how much contribution we're soliciting from users, and how often we encourage them to reach out directly with questions. Users should feel welcome to submit small fixes, feature improvements that address some deficiency in their workflow, or address an open Issue, but we don't really have the test coverage or developer bandwidth yet to open the floodgates for that full-on open source lifestyle.